models/linkages/R/split.inputs.LINKAGES.R

```diff
@@ -17,10 +17,27 @@
 ##' @return files split up climate files
 ##' @export
 ##' 
-split.inputs.LINKAGES <- function(settings, start.time, stop.time) {
+split.inputs.LINKAGES <- function(settings, start.time, stop.time, inputs) {
   
-  new.met <- paste0(settings$rundir, "/climate.Rdata")  # doesn't do anything but write stuff to README
-  settings$run$inputs$met$path <- new.met  #HACK
+  #### Load all temp and precip met for site
+  met <- inputs$met$path
+  load(met)
   
-  return(settings$run$inputs)
+  #### Add years for indexing
+  temp.mat.years <- cbind(temp.mat,seq(850,2010,1))
+  precip.mat.years <- cbind(precip.mat,seq(850,2010,1))
+  
+  #### Grab needed years
+  temp.mat <- temp.mat.years[temp.mat.years[,13]==year(start.time):year(stop.time),1:12]
+  precip.mat <- precip.mat.years[precip.mat.years[,13]==year(start.time):year(stop.time),1:12]
+  
+  #### Save to climate file
+  save(temp.mat, precip.mat, file = paste0(settings$rundir, "/climate.Rdata"))
+  
+  new.met <- paste0('',settings$rundir, "/climate.Rdata")
+  settings$run$inputs$met$path <- new.met #Do I need to change the ID too? Idk what I would change it to. 
```

Comment:
probably don't need to split the met at all -- would probably be much simpler to teach LINKAGES to read years